lib/mbta_v3_api/store/vehicles.ex

```diff
@@ -0,0 +1,198 @@
+defmodule MBTAV3API.Store.Vehicles do
```

Comment:
This looks all identical to `Store.Predictions` except for the Store class and the Impl, would it be at all straightforward to make something more generic that could just take those two params and generate the module? Or is it possible/likely that other ones will have different implementations?

Reply:
Er, well, I guess alerts are the only other realtime stream, do we plan to store those in ETS too, or just these two? If it's just going to be two then I'm more fine with the repetition, but three is pushing it.

Reply:
Yeah that's fair! I don't have plans to convert alerts as part of this ticket (but there will be a bunch of follow-up tasks, like converting the vehicles channel to use the new vehicles table & predictions by trip to use the new trips table). 

I did a time boxed pass of making this more generic in a few ways:
* macro to hide the boilerplate calling the correct module for testing
* common timed_fetch function

Predictions & Vehicles are still mostly similar, but Predictions has the extra logic for pulling in other related records. I think any further refactoring to make these common would be worth holding off on.

Reply:
That makes sense, this looks great, thank you!